### PR TITLE
Fix: made footer center aligned 

### PR DESCRIPTION
**Pull request by nlok5923**
## Issue that this pull request solves

 Closes: #566

## Proposed changes
made footer center aligned
### Brief description of what is fixed or changed
made footer center aligned
## Types of changes
style fix
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update (Documentation content changed)
- [x] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
image

## Other information
Not needed
Any other information that is important to this pull request

### CODE REVIEW

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @nlok5923 the social media icons still looks left aligned. Please fix that. Rest looks good to me :)

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @nlok5923 the UI of the footer's social icons seems distorted now. Please fix it (probably [this](https://github.com/HITK-TECH-Community/Community-Website/pull/576/files#diff-91971011745d4a36086a927f379f10618f2da07dcc5dfde7f6bd83fb7839ebcfR352) line is creating this issue).
Footer_icon

**Comment by nlok5923** [comment]
Thanks for pointing it out @Kajol-Kumari working on it.

**Comment by Kajol-Kumari** [comment]
> Thanks for pointing it out @Kajol-Kumari working on it.

Hey @nlok5923 I can still see the icon's grid distorted, can you please check it on your end and fix it?

File: frontend/src/components/Footer/footer.module.scss [hunk 1]
```diff
@@ -151,6 +152,9 @@
   color: #c2d6d7;
 }
 
+.nav-ul li {
+  text-align: center;
+
```
Comment (Kajol-Kumari) [inline frontend/src/components/Footer/footer.module.scss:157]: closing paranthesis (`}`) is missing here

Comment (nlok5923) [reply frontend/src/components/Footer/footer.module.scss:157]: yes this is the reason of build faliure.

**Pull request closed without merging**